crates/protocol/interop/src/constants.rs

```diff
@@ -1,13 +1,14 @@
 //! Constants for the OP Stack interop protocol.
 
 use alloy_primitives::{Address, address};
+use core::time::Duration;
 
 /// The address of the L2 cross chain inbox predeploy proxy.
 pub const CROSS_L2_INBOX_ADDRESS: Address = address!("4200000000000000000000000000000000000022");
 
 /// The expiry window for relaying an initiating message (in seconds).
 /// <https://specs.optimism.io/interop/messaging.html#message-expiry-invariant>
-pub const MESSAGE_EXPIRY_WINDOW: u64 = 180 * 24 * 60 * 60;
+pub const MESSAGE_EXPIRY_WINDOW: Duration = Duration::from_secs(30 * 24 * 60 * 60);
```

Comment:
seems like it makes more sense to keep this as u64, since down below we call `as_secs` to get the u64 back for doing the subtraction?

Reply:
I don't really mind either way personally.